daemon/api.go

```diff
@@ -113,6 +113,7 @@ var api = []*Command{
 	serialModelCmd,
 	systemsCmd,
 	systemsActionCmd,
+	routineConsoleConfStartCmd,
```

Comment:
(nitpick^2+bikeshed) we could use the prefix "internal" instead of "routine" as it's closer to the API path. But then we did not do either so far so either way is probably fine.

Reply:
I think I prefer routine for now, since it's kinda internal via the rest API, but it is meant to be used by things externally, so it's a bit of a weird situation